mitosheet/src/mito/types.tsx

```diff
@@ -1002,6 +1004,7 @@ export enum ActionEnum {
     Formulas_Dropdown_Custom = 'custom formulas dropdown',
     Formulas_Dropdown_More = 'more formulas dropdown',
     OpenSearch = 'open search',
+    OpenSearchAndReplace = 'open search and replace',
```

Comment:
Should these be open find, and open find and replace? And then we go update their titles in the actions as well